packages/react-core/src/components/DescriptionList/examples/DescriptionList.md

```diff
@@ -3,14 +3,25 @@ id: 'Description list'
 section: components
 cssPrefix: 'pf-c-description-list'
 beta: true
-propComponents: ['DescriptionList', 'DescriptionListDescription', 'DescriptionListGroup', 'DescriptionListTerm']
+propComponents:
+  [
+    'DescriptionList',
+    'DescriptionListDescription',
+    'DescriptionListGroup',
+    'DescriptionListTerm',
+    'DescriptionListTermHelpText',
+    'DescriptionListTermHelpTextButton',
+    'Popover',
+  ]
 ---
 
-import { Button, DescriptionList, DescriptionListTerm, DescriptionListDescription, DescriptionListGroup } from '@patternfly/react-core';
+import { Button, DescriptionList, DescriptionListTerm, DescriptionListDescription, DescriptionListGroup, DescriptionListTermHelpText, DescriptionListTermHelpTextButton, Popover } from '@patternfly/react-core';
```

Comment:
You don't need to include imports from `@patternfly/react-core` or `@patternfly/react-table` at the top of MD files anymore. Those 2 packages are automatically injected.